docs/notifications/03-showrunners-scaffold/03-Examples/08-Showrunner-Example-Time-Interval-Price-Alerts.mdx

```diff
@@ -0,0 +1,223 @@
+---
+id: docs-slider-settings-with-showrunners-example
+title: Time Interval Price Alerts
+hide_title: true
+slug: ./price-alerts-with-showrunners-example
+displayed_sidebar: pushNotificationSidebar
+sidebar_position: 7
+image: "/assets/docs/previews/docs_notifications_tutorials_push_react_component_playground_example--push_sdk_react_playground.png"
+---
+
+# Slider Notification Settings in Showrunners
+
+This example is intended to get you understand slider based notification settings with a real-world application. For the example we are going to look at a scenario where users can choose a time interval and showrunners framework will notify them as per their request. Checkout [Showrunners Docs](https://push.org/docs/notifications/showrunners-scaffold/get-started/), [Showrunners Framework](https://github.com/ethereum-push-notification-service/push-showrunners-framework/tree/main), [Channel Settings Docs](https://push.org/docs/notifications/build/create-channel-settings/) and [Channel Settings Demo](https://github.com/ethereum-push-notification-service/push-showrunners-framework/tree/main/src/sample_showrunners/bank) for better understanding!
```

Comment:
url should follow relative markdown format, ie /docs/.... also include description in "". For example:

[Showrunners Docs](/docs/notifications/showrunners-scaffold/get-started/ "Introduction to Showrunners docs")